modules/nextflow/src/main/groovy/nextflow/k8s/K8sConfig.groovy

```diff
@@ -140,6 +140,8 @@ class K8sConfig implements Map<String,Object> {
 
     String getNamespace() { target.namespace }
 
+    boolean getJob() { target.job }
```

Comment:
Can we rename the config option to `jobs` or `useJobs` so that it isn't confused with `pod` options? Also make sure to document the new option in `docs/config.rst`.

Reply:
job and jobs are too similiar, so `useJobs` then?

Reply:
sure